dev-requirements.txt

```diff
@@ -1,3 +1,4 @@
+# This is only meant for Scalyr's internal development and dev environment. These dependencies are not needed to run the tool
```

Comment:
Can you specify what tool this is for?  I.e., this is formated for tools like `blah`.